spec/purpose_and_scope.md

```diff
@@ -369,6 +369,31 @@ recommended not to add other functions or objects, because that may make it
 harder for users to write code that will work with multiple array libraries.
 
 
+### Discoverability
+
```

Comment:
This is a bit cryptic for a reader, like why does it exist? what goes wrong if an entrypoint is not implemented? The one word _may_ is not enough, I'll try to add a note with rationale. Especially because this is in the "Purpose and scope" section, that is important.

Reply:
Thanks!